src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -134,33 +157,36 @@ private boolean isExceptionRecordCreated(UUID randomPoBox) {
             )
         );
 
-        exceptionRecordCase = caseDetailsList.isEmpty() ? null : caseDetailsList.get(0);
-        return exceptionRecordCase != null;
+        exceptionRecord = caseDetailsList.isEmpty() ? null : caseDetailsList.get(0);
+        return exceptionRecord != null;
     }
 
-    private void verifyExistingCaseIsUpdatedWithExceptionRecordCaseDetails() {
+    private Boolean isExceptionRecordAttachedToTheCase(int expectedScannedDocsSize) {
+
         CaseDetails updatedCase = caseRetriever.retrieve(
-            existingCase.getJurisdiction(),
-            String.valueOf(existingCase.getId())
+            caseDetails.getJurisdiction(),
+            String.valueOf(caseDetails.getId())
         );
 
         List<ScannedDocument> updatedScannedDocuments = getScannedDocumentsForSupplementaryEvidence(updatedCase);
-
-        List<ScannedDocument> scannedDocuments = getScannedDocumentsForExceptionRecord(exceptionRecordCase);
+        List<ScannedDocument> scannedDocuments = getScannedDocumentsForExceptionRecord(exceptionRecord);
 
         assertThat(scannedDocuments).isNotEmpty();
-        assertThat(scannedDocuments.size()).isEqualTo(1);
-
         assertThat(updatedScannedDocuments).isNotEmpty();
-        assertThat(updatedScannedDocuments.size()).isEqualTo(1);
 
-        ScannedDocument updatedCaseEvidence = updatedScannedDocuments.get(0);
-        ScannedDocument caseEvidence = scannedDocuments.get(0);
+        return updatedScannedDocuments.size() == expectedScannedDocsSize
+            && updatedScannedDocuments.containsAll(scannedDocuments);
```

Comment:
so the size parameter is not actually needed here since we do a contains all .
why not just `updatedScannedDocuments.size() == scannedDocuments.size` ?

Reply:
updated the assertions now